src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -242,12 +244,12 @@ void CascadiaSettings::_ResolveDefaultProfile()
 //   warnings if we failed to find the default.
 void CascadiaSettings::_ValidateDefaultProfileExists()
 {
-    const auto defaultProfileGuid = GlobalSettings().DefaultProfile();
-    const bool nullDefaultProfile = defaultProfileGuid == GUID{};
+    const auto defaultProfileGuid = winrt::guid(GlobalSettings().DefaultProfile());

```

Comment:
please don't use `()` for constructor calls or casting.

Also,

```suggestion
    const winrt::guid defaultProfileGuid{ GlobalSettings().DefaultProfile() };
```